phageAPI/util/tests.py

```diff
@@ -0,0 +1,25 @@
+from unittest import TestCase
```

Comment:
Import not used.

Please rename this file to `tests/test_util.py` or similar. Tests are not being run here.